tests/test_requests.py

```diff
@@ -88,17 +89,21 @@ async def stub(req):
 
 @pytest.mark.asyncio
 @pytest.mark.parametrize("method_name", ["request", "get", "post", "execute"])
-async def test_http_client_allow_status(async_mock, client_with_status, method_name):
+async def test_http_client_allow_status(
+    async_mock, client_with_status, method_name
+) -> None:
     client = HttpClient(async_mock)
 
     # Simulate 500 Internal Server Error responses
     client._request_downloader = client_with_status(500)
 
     method = getattr(client, method_name)
 
-    url_or_request = "url"
+    url_or_request: Union[str, HttpRequest] = "url"
     if method_name == "execute":
-        url_or_request = HttpRequest(url_or_request)
+        # Similar mypy issue as noted in
+        # https://github.com/zytedata/zyte-common-items/blob/main/tests/test_mypy.py#L37-L44
```

Comment:
This link may become outdated if zyte-common-items code changes. Could you please either copy-paste some content from this link, or add a commit hash?

Reply:
Removed the link and copy-pasted the contents directly instead. https://github.com/scrapinghub/web-poet/pull/78/commits/62edbb9475a2ea6994bc5b6ab3140407de477e52